mkdocs.yml

```diff
@@ -76,6 +76,7 @@ nav:
     - Router: 'RouterConfiguration.md'
     - "Route Request": 'RouteRequest.md'
     - "Realtime Updaters": 'UpdaterConfig.md'
+    - "Migrating Configuration": 'Migrating-Configuration.md'
```

Comment:
The menu item is not DRY, it repeat the parent section. Maybe: _Migrating between versions/builds_